src/core/vectorlayercounterpart.js

```diff
@@ -1,4 +1,4 @@
-goog.provide('olcs.core.OlLayerPrimitive');
+goog.provide('olcs.core.VectorLayerCounterpart');
```

Comment:
I don't think it is misleading to have `Primitive` in the class name personally. It's a matter of taste but I find the name `VectorLayerCounterpart` ugly :) And I find it inexplicit as well. I'd like `olcs.core.PrimitiveWrapper` better than `VectorLayerCounterpart`.